assets/dfn-panel.css

```diff
@@ -1,39 +1,42 @@
 /* dfn popup panel that list all local references to a dfn */
+/**
+ * TODO: Revert changes due to https://github.com/w3c/respec/pull/2888 when
+ * https://github.com/w3c/css-validator/pull/111 is fixed.
+ */
 dfn {
   cursor: pointer;
 }
 
 .dfn-panel {
-  --fill: #fff;
   position: absolute;
-  left: var(--left); /* set via JS */
-  top: var(--top); /* set via JS */
   z-index: 35;
   min-width: 300px;
   max-width: 500px;
   padding: 0.5em 0.75em;
   margin-top: 0.6em;
   font: small Helvetica Neue, sans-serif, Droid Sans Fallback;
-  background: var(--fill);
+  background: #fff;
   color: black;
   box-shadow: 0 1em 3em -0.4em rgba(0, 0, 0, 0.3),
     0 0 1px 1px rgba(0, 0, 0, 0.05);
   border-radius: 2px;
 }
-
 /* Triangle/caret */
-.dfn-panel:not(.docked)::before,
-.dfn-panel:not(.docked)::after {
+.dfn-panel:not(.docked) > .caret {
+  position: relative;
+  top: -0.25em;
+}
+.dfn-panel:not(.docked) > .caret::before,
+.dfn-panel:not(.docked) > .caret::after {
   content: "";
   position: absolute;
   border: 10px solid transparent;
   border-top: 0;
-  border-bottom: 9px solid #a2a9b1; /* triangle outline */
-  top: -9px;
-  left: calc(var(--caret-offset, 0px) + 0.75em); /* set via JS */
```

Comment:
Might be better to use `attr()` in the future rather than just reverting this. Google just posted an intent to implement for the feature. https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/FGCgsKmylhw/BSUrk2roCQAJ

Since the validator is quite inactive, the browser implementation might be quicker.

Reply:
Looking at the code, the validator already supports `attr()` so we probably can just use it when the browser support lands.

Reply:
Even though nuHTML allows advanced `attr()`, [CSS validator](http://jigsaw.w3.org/css-validator/validator) doesn't.

Reply:
Ah right, it can parse the syntax but the validation is broken. 🤔

Reply:
Hey, actually it does validate it!

```
.abc {
  width: calc(attr(data-foo px) / 10);
}
```

Try this code.

Reply:
Oh right! I missed the `px` suffix